lib/lotus/generators/action.rb

```diff
@@ -108,6 +114,14 @@ def assert_action!
         end
       end
 
+      # @since x.x.x
+      # @api private
+      def assert_http_method!
+        if !HttpRouter::Route::VALID_HTTP_VERBS.include?(_http_method.upcase)
```

Comment:
IMHO `HttpRouter` is low-level, there should be a public interface in `Lotus::Router` that we should use, please check